website/src/utils/Utils.jsx

```diff
@@ -42,4 +42,29 @@ export const errorApi = 'An error occurred while retrieving data from the API. P
 
 //NUMBER OF PIXELS TO OPEN AND CLOSE THE DROP-DOWN
 export const openDropDownValue = 1000
-export const closeDropDownValue = 58
\ No newline at end of file
+export const closeDropDownValue = 58
+
+ // OPEN DROP DOWN
+
+ export const openDropDown = (openDropDown, setOpenDropDown) =>{
+    if (openDropDown === closeDropDownValue) {
+        setOpenDropDown(openDropDownValue);
+      } else {
+        setOpenDropDown(closeDropDownValue);
+      }
+}
```

Comment:
If I understand properly this function inverses the value of the drop down using `setOpenDropDown`. Meaning that if it is closed, we open it, and vice versa. It would be helpful to rename the first argument of your function (`openDropDown`) to something slightly more understandable such as `currentValue`.